lib/config/foldertype.go

```diff
@@ -30,12 +30,12 @@ func (t FolderType) MarshalText() ([]byte, error) {
 
 func (t *FolderType) UnmarshalText(bs []byte) error {
 	switch string(bs) {
-	case "readwrite":
-		*t = FolderTypeReadWrite
-	case "readonly":
-		*t = FolderTypeReadOnly
+	case "readwrite", "sendreceive":
+		*t = FolderTypeSendReceive
+	case "readonly", "sendonly":
+		*t = FolderTypeSendOnly
```

Comment:
Ah ok, this might work actually, instead of the magic extra struct, the question is however when do we remove these old values, I suspect never :/ or we could remove them if we did the extra struct migration, but then the struct would have to live forever. However then the code to deal with legacy stuff is nicely living in coversion section opose an unmarshal method of a random type. @calmh any strong preferences?